src/main/java/james/altoclef/taskcreator/graphics/AltoFrame.java

```diff
@@ -319,10 +317,12 @@ public void addMenu() {
         JMenu fileMenu = new JMenu("File");
         JMenuItem fileMenu_newFile = new JMenuItem("New");
         JMenu fileMenu_subMenuOpen = new JMenu("Open");
-        JMenuItem fileMenu_subMenuOpen_fromFileSystem = new JMenuItem("From File System");
+        JMenuItem fileMenu_subMenuOpen_fromFileSystem = new JMenuItem("From File chooser");
```

Comment:
Let's change the word "File Chooser" to "Select a File", that way "From File Path (fix capitalization)" makes more sense.